scripts/sentry-cli.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
damn.. wish we could just like, use python for this type of stuff

Reply:
We can move in that direction if we wish to (and I would be willing to) since I find shell has too much magic and difficult syntax to learn.

We would also get performance tracking via Python SDK.